src/lib/browser/BrowserPage.ts

```
@@ -668,6 +670,224 @@ export class BrowserPage {
+      // Page down for next shot
+      await this.sendKeys('PageDown')
+      await new Promise(resolve => setTimeout(resolve, BrowserPage._PDF_SCROLL_DELAY_MS))
+    }
+    return images
+  }
+
+  // Resolve original PDF URL if viewing through Chrome's PDF viewer (src param)
+  private async _resolvePdfSourceUrl(): Promise<string> {
+    try {
+      const tab = await chrome.tabs.get(this._tabId);
+      let currentUrl = tab.url || this._url || '';
```

ESLint

[error] 'currentUrl' is never reassigned. Use 'const' instead.

(prefer-const)